package.json

```diff
@@ -32,15 +32,15 @@
     "karma-script-launcher": "^1.0.0",
     "karma-spec-reporter": "^0.0.32",
     "karma-webpack": "^4.0.2",
+    "portfinder-sync": "0.0.2",
     "redux-test-utils": "^1.0.2",
     "style-loader": "^2.0.0",
     "svg-url-loader": "^7.1.1",
     "to-string-loader": "^1.1.6",
     "webpack": "^4.46.0",
     "webpack-bundle-analyzer": "^4.4.0",
     "webpack-cli": "^4.6.0",
-    "webpack-dev-server": "^3.11.2",
-    "portfinder-sync": "0.0.2"
+    "webpack-dev-server": "^3.11.2"
```

Comment:
I don't know why this is was modified 🤪
Please have a look at every file before committing it...